src/components/page/WithGATracker/index.tsx

```diff
@@ -10,6 +10,19 @@ import store from '../../../store';
 type Props = RouteComponentProps<RouteParams>;
 let username = (getUser(store.getState()) || {}).username || '';
 
+/**
+ * helper function to get the Google Analytics tracking code
+ * @param {FlexObject|undefined} options tracking options for the page view
+ * @returns {string}
+ */
+export const getGaCode = (options: FlexObject | undefined) => {
+  const gaCode =
+    (options && !!!options.GA_CODE && options.GA_CODE) ||
+    ((!options || !!options.GA_CODE) && GA_CODE) ||
+    '';
+  return gaCode;
+};
+
```

Comment:
This conditional checks the existing optional `options` param for `GA_CODE` and uses that instead of `env.GA_CODE`.